src/probnum/diffeq/odefiltsmooth/diffusions.py

```diff
@@ -0,0 +1,120 @@
+"""Diffusion calibrations.
+
+Maybe move this to `statespace` or `filtsmooth` in the future, but for
+now, it is well placed in `diffeq`.
+"""
+
+
+import abc
+from typing import Union
+
+import numpy as np
+
+from probnum import random_variables
+from probnum.type import FloatArgType
+
+DiffusionType = Union[FloatArgType]
+"""Acceptable diffusion types are -- in principle -- floats and arrays of shape (d,).
+In other words, everything that behaves well with `cov1 + diffusion*cov2` and
+`chol1 + sqrt(diffusion) * chol2`.
+For now, let's only use floats, because it is not clear to me how to best implement the square-root behaviour.
+"""
+
+
+class Diffusion(abc.ABC):
+    r"""Interface for diffusion models :math:`\sigma: \mathbb{R} \rightarrow \mathbb{R}^d` and their calibration."""
+
+    def __repr__(self):
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def __call__(self, t) -> DiffusionType:
+        """Evaluate the diffusion."""
+        raise NotImplementedError
+
+    @abc.abstractmethod
+    def __getitem__(self, idx):
+        """Get the i-th diffusion from the list."""
+        pass
+
+    def calibrate_locally(self, meas_rv):
```

Comment:
I think it is best to keep this function name rather high level and not talk about quasi-MLEs here -- who knows where we are heading.

Reply:
`estimate_locally`